docs/en/compute/AWS-EMR.md

```diff
@@ -152,16 +152,85 @@ Tuning of Alluxio properties can be done in a few different locations.
+                    upon to restore cluster state after a previous shutdown.
+                    
+
+  -d                An s3:// or http(s):// URI which points to an Alluxio
+                    tarball. This script will download and untar the
+                    Alluxio tarball and install Alluxio at /opt/alluxio if an
+                    Alluxio installation doesn't already exist at that location.
+                    
+
+  -f                An s3:// or http(s):// URI to any remote file. This property
+                    can be specified multiple times. Any file specified through
+                    this property will be downloaded and stored with the same
+                    name to /opt/alluxio/conf/
+                    
+
+  -i                An s3:// or http(s):// URI which represents the URI of a
+                    previous Alluxio journal backup. If supplied, the backup
+                    will be downloaded, and upon Alluxio startup, the Alluxio
+                    master will read and restore the backup.
+                    
+
+  -p                A string containing a delimited set of properties which
+                    should be added to the
+                    ${ALLUXIO_HOME}/conf/alluxio-site.properties file. The
+                    delimiter by default is a semicolon ";". If a different
+                    delimiter is desired use the [-s] argument.
+                    
+
+  -s                A string containing a single character representing what
+                    delimiter should be used to split the Alluxio properties
+                    provided in the [-p] argument.
+```
+
 ### Alluxio Service
 
-Any server-side configuration changes must be made in the `alluxio-emr.sh` bootstrap script.
-In the section for generating the `alluxio-site.properties`, add a line with the configuration
-needed to append to the bottom of the file.
-Options can also be passed as the 3rd argument to the bootstrap script with a ';' delimiter.
+Making configuration changes to the Alluxio Service can be done in a few different ways via the
+bootstrap script.
+The [-p] flag allows users to pass in a set of delimited key-value properties to be set on all of
+the Alluxio nodes. An alternative would be to pass in a custom file using the [-f] flag named
+`alluxio-site.properties`.
+The bootstrap will make sure to overwrite any user-provided configs while retaining any defaults
+that are not overwritten.
+The bootstrap also allows users to install previous versions of Alluxio (>=2.0) by specifying
+a download URL (HTTP or S3 only).
 
 ### Alluxio Client
 
 Generic client-side properties can also be edited via the bootstrap script as mentioned above.
 This is mostly for the native client (CLI).
 Property changes for a specific service like Presto/Hive should be done in the respective
-configuration file i.e. `core-site.xml`, `hive.catalog`.
+configuration file i.e. `core-site.xml`, `hive.catalog` via the EMR JSON entry-point.
```

Comment:
is entry-point the right word here?

Reply:
I think it's fine. Do you have a better suggestion?